bin/kano-sync

```diff
@@ -67,58 +69,77 @@ def do_login():
     login_success, value = login_using_token()
     if not login_success:
         logger.debug('token login not successful msg: {}'.format(value))
-        if args.silent:
+        if silent:
             logger.debug('exiting because of --silent')
-            sys.exit()
+            return False
         else:
```

Comment:
As we return above, can we remove the else and a level of indentation here (b the w, I do realise that this wasn't your doing but I have just noticed it ;-))